packages/browser-destinations/destinations/fullstory/src/__tests__/fullstory.test.ts

```diff
@@ -236,3 +194,93 @@ describe('#identify', () => {
     )
   })
 })
+
+describe('#page', () => {
+  it('sends page events to fullstory on "page" (category edition)', async () => {
+    const [, , viewed] = await fullstory({
+      orgId: 'thefullstory.com',
```

Comment:
Nit (no strong feelings about this): Wonder if it's worth using a fake org id given this code ends up in Segment's publicly visible OSS repo.

Reply:
+1